### PR TITLE
Skip a flaky e2e spec

### PR DESCRIPTION
**Pull request by maestromac**
## What type of PR is this? (check all applicable)
- [x] DX

## Description
Temporarily disable flaky spec

## Added tests?
- [x] no, because they aren't needed

## Added to documentation?
- [x] no documentation needed

## [optional] Are there any post deployment tasks we need to perform?
n/a

### CODE REVIEW

File: cypress/integration/articleFlows/commentOnArticle.spec.js [hunk 1]
```diff
@@ -202,7 +202,7 @@ describe('Comment on articles', () => {
       cy.findByText('@search_user_1').should('not.be.visible');
     });
 
-    it('should reply to a comment with user mention autocomplete', () => {
+    xit('should reply to a comment with user mention autocomplete', () => {
```
Comment (juliannatetreault) [inline cypress/integration/articleFlows/commentOnArticle.spec.js:205]: Should we leave a comment or TODO here explaining why it's skipped for now?

**Review by juliannatetreault — APPROVED** [review]
Left one, quick question, otherwise 🚀

**Review by djuber — APPROVED** [review]
we probably _do_ want a comment documenting that we intend to revisit this, but non-blocking.

File: cypress/integration/articleFlows/commentOnArticle.spec.js (hunk 1 above)
Comment (nickytonline) [reply cypress/integration/articleFlows/commentOnArticle.spec.js:205]: I'm going to revisit this later today/tomorrow.

**Review by nickytonline — APPROVED** [review]

File: cypress/integration/articleFlows/commentOnArticle.spec.js (hunk 1 above)
Comment (maestromac) [inline cypress/integration/articleFlows/commentOnArticle.spec.js:205]:
```suggestion
    // TODO: Flaky spec
    xit('should reply to a comment with user mention autocomplete', () => {
```

**Pull request merged**